registry/base_configuration_listener.go

```diff
@@ -60,7 +60,7 @@ func (bcl *BaseConfigurationListener) InitWith(key string, listener config_cente
 	}
 }
 
-// Process ...
+// Process can reference ConfigurationListener.Process
```

Comment:
"Process can reference ConfigurationListener.Process" I can not get what the meaning of it.